hack/generator/pkg/jsonast/jsonast.go

```diff
@@ -552,7 +543,7 @@ func allOfHandler(ctx context.Context, scanner *SchemaScanner, schema *gojsonsch
 		case *astmodel.ResourceType:
 			// it is a little strange to merge one resource into another with allOf,
 			// but it is done and therefore we have to support it.
-			// (an example is Microsoft.VisualStudio’s Project type)
+			// (an example is Microsoft.VisualStudio's Project type)
```

Comment:
I can get rid of the disagreement ...
```suggestion
			// (an example is the Microsoft.VisualStudio Project type)
```